### ISSUE TITLE
Completions should allow for multi-selection

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#1611]
E.g. for `:tabclose`, it would be handy to be able to mark multiple tabs for closure simultaneously.

**Comment by Woastscho** [comment]
Is there any solution in 2024 ?

**Comment by bovine3dom** [comment]
Not exactly. I want to rewrite all the completions from scratch and it's something I would like to add then. I started here https://github.com/tridactyl/tridactyl/pull/3895 but lost motivation.

There is ex.execute_ex_on_completion which is bound to ctrl+enter by default which allows you to execute a completion and keep the command line open.

see `:viewconfig exmaps` to see all the default binds
